iina/MainWindowController.swift

```diff
@@ -1657,7 +1657,7 @@ class MainWindowController: PlayerWindowController {
   }
 
   private func showUI() {
-    if player.disableUI { return }
+    guard !player.disableUI, ![.willShow, .shown].contains(animationState) else { return }
```

Comment:
I'm confused about why this change is needed and how it relates to the CPU issue.

Reply:
This function is executed a lot of times when the cursor is moved even though it is not necessary. This is not related to the timer issue.